Resources/Prototypes/_Goobstation/Voice/speech_emotes.yml

```
@@ -121,3 +121,25 @@
     - ultra farts
     - ultra farted
     - ultra farting
+
+- type: emote 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)